src/mmx_node.cpp

```diff
@@ -71,6 +72,7 @@ int main(int argc, char** argv)
 
 	vnx::log_info() << "AVX2 support:   " << (avx2_available() ? "yes" : "no");
 	vnx::log_info() << "SHA-NI support: " << (sha256_ni_available() ? "yes" : "no");
+	vnx::log_info() << "ARM-SHA2 support: " << (sha256_arm_available() ? "yes" : "no");
```

Comment:
Could probably surround with an #ifdef so it don't show on x86.

Reply:
Fixed, uploaded Rev02.